src/routes/OrganizationsList/CreateOrganizationModal.tsx

```diff
@@ -14,21 +14,55 @@ import FormError from 'src/components/errors/FormError';
 import {addDisplayError} from 'src/resources/ErrorHandling';
 import {userRefreshOrgList} from 'src/hooks/UseRefreshPage';
 
+interface Validation {
+  message: string;
+  isValid: boolean;
+  type: 'default' | 'error' | 'warning';
+}
+
+const defaultMessage: Validation = {
+  message:
+    'This will also be the namespace for your repositories. Must be alphanumeric, all lowercase, at least 2 characters long and at most 255 characters long',
+  isValid: true,
+  type: 'default',
+};
+
 export const CreateOrganizationModal = (
   props: CreateOrganizationModalProps,
 ): JSX.Element => {
   const [organizationName, setOrganizationName] = useState('');
   const [organizationEmail, setOrganizationEmail] = useState('');
   const [invalidEmailFlag, setInvalidEmailFlag] = useState(false);
-  const [validOrgName, setValidOrgName] = useState(true);
+  const [validation, setValidation] = useState<Validation>(defaultMessage);
   const [err, setErr] = useState<string>();
   const refresh = userRefreshOrgList();
 
   const handleNameInputChange = (value: any) => {
     const regex = /^([a-z0-9]+(?:[._-][a-z0-9]+)*)$/;
-    setValidOrgName(
-      regex.test(value) && value.length < 256 && value.length >= 2,
-    );
+    if (!regex.test(value) || value.length >= 256 || value.length < 2) {
+      setValidation({
+        message:
+          'Must be alphanumeric, all lowercase, at least 2 characters long and at most 255 characters long',
+        isValid: false,
+        type: 'error',
+      });
+    } else if (value.length > 30 || value.length < 4) {
+      setValidation({
+        message:
+          'Namespaces less than 4 or more than 30 characters are only compatible with Docker 1.6+',
+        isValid: true,
+        type: 'warning',
+      });
+    } else if (value.includes('.') || value.includes('-')) {
+      setValidation({
+        message:
+          'Namespaces with dashes or dots are only compatible with Docker 1.9+',
```

Comment:
this warning did not show up for me. It's probably getting past the regex check

Reply:
do you have the name you used? I can test it out to see what the issue is.

Reply:
huh, I seems to be working now. Initially I didn't see the warning for any name that had a `.` or `-`